packages/amplify-graphql-api-construct-tests/src/utils/auth.ts

```diff
@@ -114,3 +114,34 @@ export const signInCognitoUser = async (options: SignInCognitoUserInput): Promis
     username,
   };
 };
+
+export const getUsernameClaimFromJwt = (accessToken: string): string => {
```

Comment:
Added these for an aborted foray into owner==primary key testing. We don't actually support that case well, but I've left the utilities for upcoming cases.